.gitignore

```diff
@@ -7,6 +7,7 @@ yarn-error.log*
 lerna-debug.log*
 .idea
 .vsc
+package-lock.json
```

Comment:
**nitpick:** It should not be gitignored. I will check why Nerdy still tries to analyze lock file